tests/inferences/test_renyi_divergence.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import edward as ed
+import numpy as np
+import tensorflow as tf
+
+from edward.models import Bernoulli, Normal
+from edward.inferences.renyi_divergence import RenyiDivergence
+
+
+class test_renyi_divergence_class(tf.test.TestCase):
+
+  def _test_normal_normal(self, Inference, *args, **kwargs):
```

Comment:
Since `RenyiDivergence` is used across all tests, you don't need it as an arg to the test functions.

Reply:
I've used the same template as `test_klpq` where only `KLpq` is used during the tests and where `Inference` is stil an argument to the test functions.
But I think I have modified to be closer to what you had in mind